Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/Snapshots/ComplexCalculator_Add_Generated.g.cs

```diff
@@ -33,5 +35,24 @@ public Amazon.Lambda.APIGatewayEvents.APIGatewayHttpApiV2ProxyResponse Add(Amazo
                 StatusCode = 200
             };
         }
+
+        private static void SetExecutionEnvironment()
+        {
+            const string envName = "AWS_EXECUTION_ENV";
+            const string amazonLambdaAnnotations = "amazon-lambda-annotations";
+            const string sourceGeneratorVersion = "0.1.0.0";
+
+            var envValue = new StringBuilder();
+
+            // If there is an existing execution environment variable add the annotations package as a suffix.
+            if(!string.IsNullOrEmpty(Environment.GetEnvironmentVariable(envName)))
+            {
+                envValue.Append($"{Environment.GetEnvironmentVariable(envName)}_");
+            }
+
+            envValue.Append($"{amazonLambdaAnnotations}_{sourceGeneratorVersion}");
```

Comment:
Since this is generated why not generated with the values actually in the Append call to avoid doing the string interpolation. That also means the lines 42 and 43 disappear and this line looks like

```
envValue.Append("amazon-lambda-annotations_0.1.0.0");
```